cli/src/commands/debug.rs

```diff
@@ -94,8 +98,12 @@ pub enum DebugOperationDisplay {
 /// List the recursive entries of a tree.
 #[derive(clap::Args, Clone, Debug)]
 pub struct DebugTreeArgs {
-    #[arg(long, short = 'r', default_value = "@")]
-    revision: RevisionArg,
+    #[arg(long, short = 'r')]
+    revision: Option<RevisionArg>,
+    #[arg(long, conflicts_with = "revision")]
+    id: Option<String>,
+    #[arg(long, requires = "id")]
+    dir: Option<String>,
```

Comment:
I think this could work with `-r` in theory.

If you agree, perhaps leave a comment to this effect?

Reply:
I could be combined, but I don't see a use case for it since we also support filtering by paths.

I see the dir+id combo as an alternative way of looking up a tree. With the git backend, the directory is not used by the backend, but for other backends it can be, so that's why the user needs to pass it.